src/app/pages/single-course-page/CoursePage.tsx

```diff
@@ -0,0 +1,174 @@
+import { AxiosError } from "axios";
+import Hls from "hls.js";
+import { useEffect, useState } from "react";
+import { useDispatch } from "react-redux";
+import { useNavigate } from "react-router-dom";
+import {
+  coursePageSlice,
+  coursesListPageSlice,
+  useAppSelector,
+} from "../../redux";
+import { Client, IGetCourse } from "../../services";
+import { authErrorResponse } from "../../services/controllers/constants";
+import { Title, Wrapper, Description, BackButton } from "./styles";
```

Comment:
When importing more than two elements from a single directory, it is better to structure them like this way:

```suggestion
import {
  Title,
  Wrapper,
  Description,
  BackButton,
} from "./styles";
```

Reply:
Thanks for suggestion! Firstly, it was formatted such way by default linter, what means it is not bad practice. Secondly, in my opinion, it is more about "coding taste", because as for me such inline code is more readable.

Reply:
Я схиляюсь до думки, що такі речі не потрібно робити вручну. Подібне форматування повинне виконуватись лінтером/претіером автоматично, і якщо лінтер це не виправив, то значить це або не потрібно, або у вас недоналаштований лінтер)
І наскільки я знаю, зазвичай подібні переноси застосовуються у лінтері не по правилу "більше двох імпортів", а коли довжина строки імпорту починає перевищувати рекомендовану